{"uid":"7cab27e631aff5d2","name":"test_register[result0]","fullName":"testcases.test_user#test_register","historyId":"db808937b6c6b4b8ae95bf2b74bb3853","time":{"start":1706174785104,"stop":1706174785123,"duration":19},"status":"failed","statusMessage":"AssertionError: assert 0 == 1001","statusTrace":"result = ('004', 'succ-register', 'user', '192.168.0.105:3000', '/api/user/register', 'post', ...)\n\n    @pytest.mark.parametrize(\"result\", data_register())\n    def test_register(result):\n        id = result[0]\n        name = result[1]\n        module = result[2]\n        request_prefix = result[3]\n        request_url = result[4]\n        request_type = result[5]\n        need_login = result[6]\n        data = result[7]\n        expect_result = result[8]\n        real_result = result[9]\n        test_result = result[10]\n        data_type = result[11]\n        url = 'http://' + request_prefix + request_url\n        dataDict = json.loads(data)\n        expect_resultDict = json.loads(expect_result)\n        # print( url, dataDict, type(dataDict),expect_resultDict, type(expect_resultDict))\n        res = Request.send_request(url, method=request_type, data=dataDict).json()\n        # print('res==>',res)\n>       assert  expect_resultDict[\"errno\"]==res[\"errno\"]\nE       assert 0 == 1001\n\ntestcases\\test_user.py:82: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":3,"retriesStatusChange":true,"beforeStages":[],"afterStages":[],"labels":[{"name":"parentSuite","value":"testcases"},{"name":"suite","value":"test_user"},{"name":"host","value":"DESKTOP-061EAN7"},{"name":"thread","value":"15864-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"testcases.test_user"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"result","value":"('004', 'succ-register', 'user', '192.168.0.105:3000', '/api/user/register', 'post', '0', '{\"username\":\"test222333\",\"password\":\"test222333\",\"confirm_password\": \"test222333\"}', '{\"errno\":0,\"data\":{},\"message\":\"注册成功\"}', '{\\n    \"errno\": 0,\\n    \"data\": {\\n        \"username\": \"test222333\",\\n        \"_id\": \"6599251dba8c71a12392c891\",\\n        \"createdAt\": \"2024-01-06T10:02:05.636Z\",\\n        \"updatedAt\": \"2024-01-06T10:02:05.636Z\",\\n        \"__v\": 0\\n    },\\n    \"message\": \"注册成功\"\\n}', 'miss', 'miss')"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"9802d0da6fbd139f","status":"broken","statusDetails":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='192.168.0.105', port=3000): Max retries exceeded with url: /api/user/register (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002356620F730>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1706024569517,"stop":1706024571536,"duration":2019}},{"uid":"8862ef4f323535cf","status":"failed","statusDetails":"AssertionError: assert 0 == 1001","time":{"start":1705648175670,"stop":1705648175678,"duration":8}},{"uid":"777ee1a42b2d201a","status":"failed","statusDetails":"AssertionError: assert 0 == 1001","time":{"start":1705648156897,"stop":1705648159509,"duration":2612}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"7cab27e631aff5d2.json","parameterValues":["('004', 'succ-register', 'user', '192.168.0.105:3000', '/api/user/register', 'post', '0', '{\"username\":\"test222333\",\"password\":\"test222333\",\"confirm_password\": \"test222333\"}', '{\"errno\":0,\"data\":{},\"message\":\"注册成功\"}', '{\\n    \"errno\": 0,\\n    \"data\": {\\n        \"username\": \"test222333\",\\n        \"_id\": \"6599251dba8c71a12392c891\",\\n        \"createdAt\": \"2024-01-06T10:02:05.636Z\",\\n        \"updatedAt\": \"2024-01-06T10:02:05.636Z\",\\n        \"__v\": 0\\n    },\\n    \"message\": \"注册成功\"\\n}', 'miss', 'miss')"]}